theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -425,6 +425,9 @@ def test_pooling3d():
     mode_without_gpu2 = mode_without_gpu.including()
     mode_without_gpu2.check_isfinite = False
```

Comment:
Does the part about not checking for infinity still apply? We could add it to the new mode.